test/fuzzing/README.md

```diff
@@ -0,0 +1,30 @@
+# Fuzzing umoci
+
+Umoci has a series of fuzz tests. These are implemented by way of [go-fuzz](https://github.com/dvyukov/go-fuzz).
+
+## Running the fuzzers
+
+To run the fuzzers, first build a container from the Docker image in `umoci/test/fuzzing`:
+
+```bash
+sudo docker build -t umoci-fuzz .
+```
+Next, get a shell in the container:
+```bash
+sudo docker run -it umoci-fuzz
+```
+At this point, you can navigate to any directory that has a fuzzer and build it:
+
+```bash
+cd $PATH_TO_FUZZER
+go-fuzz-build -libfuzzer -func=FUZZ_NAME && \
+clang -fsanitize=fuzzer PACKAGE_NAME.a -o fuzzer
```

Comment:
Is it possible to make this a script which is run on `docker build` so that users don't need to find out where the fuzzers live.

Reply:
Yes, I have an idea to do that. Might I suggest we do it in a separate PR, as it might require a few modifications to the existing fuzzers.

Reply:
Sounds good, thanks!